uui/components/overlays/__tests__/Modals.test.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, { ReactNode, useState } from 'react';
 import { ModalBlocker, ModalHeader, ModalFooter, ModalWindow } from '../Modals';
-import { fireEvent, renderSnapshotWithContextAsync, screen, setupComponentForTest } from '@epam/uui-test-utils';
-import { IModal, ModalBlockerProps, useUuiContext } from '@epam/uui-core';
+import { fireEvent, getDefaultUUiContextWrapper, renderSnapshotWithContextAsync, renderWithContextAsync, screen, setupComponentForTest } from '@epam/uui-test-utils';
+import { IModal, ModalBlockerProps, useArrayDataSource, useUuiContext } from '@epam/uui-core';
 import { Button } from '../../buttons';
 import { Modals } from '@epam/uui-components';
+import { PickerInput } from '../../pickers';
 
-function TestElement(props?: ModalBlockerProps) {
+function TestElement(props: ModalBlockerProps) {
```

Comment:
There was a TypeScript error, that `props` can be `undefined` when using `...props` or `props.abort`.
Judging by the component's usage (in `handleModalOpening` function below), the props can't be optional, so I removed the optional flag.